src/schemas/ui-view-schema.json

```diff
@@ -324,9 +331,16 @@
                                       "type": {
                                         "description": "A JavaScript regular expression to filter activity types to display in this layer",
                                         "type": "string"
```

Comment:
Similarly we should probably remove `type` from here and the TypeScript type since it is no longer used.